portable/MSVC-MingW/portmacro.h

```diff
@@ -143,7 +142,18 @@ void vPortExitCritical( void );
 
 /* BitScanReverse returns the bit position of the most significant '1'
  * in the word. */
-        #define portGET_HIGHEST_PRIORITY( uxTopPriority, uxReadyPriorities )    _BitScanReverse( ( DWORD * ) &( uxTopPriority ), ( uxReadyPriorities ) )
+#if defined( __x86_64__ ) || defined( _M_X64 )
+    #define portGET_HIGHEST_PRIORITY( uxTopPriority, uxReadyPriorities ) \
+        do                                                               \
+        {                                                                \
+            DWORD ulTopPriority;                                         \
+            _BitScanReverse64( &ulTopPriority, ( uxReadyPriorities ) );  \
+            uxTopPriority = ulTopPriority;                               \
+        } while( 0 )
+#else
+    #define portGET_HIGHEST_PRIORITY( uxTopPriority, uxReadyPriorities )    _BitScanReverse( ( DWORD * ) &( uxTopPriority ), ( uxReadyPriorities ) )
+#endif
```

Comment:
@josesimoes Minor style issue - would you indent this whole block two tab's worth of spaces (should be 8 spaces I think).  That would match the existing style and current depth of the conditionals.

Reply:
Just committed those changes. I believe its correct. If not, let me know.

Reply:
@josesimoes Apologies - I am just now seeing this comment.  The indenting still isn't quite right.  Please apply the attached.
[indent-suggestion.patch](https://github.com/FreeRTOS/FreeRTOS-Kernel/files/14073231/indent-suggestion.patch)